### PR TITLE
[luci/pass] Extract FP16

### PR DESCRIPTION
**Pull request by seanshpark**
This will extract external FP16 to separate header.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
related #9488, from draft #9524

File: compiler/luci/pass/src/helpers/FP16.h [hunk 1]
```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (jinevening) [inline compiler/luci/pass/src/helpers/FP16.h:2]:
```suggestion
 * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```

**Comment by jinevening** [comment]
Not for this PR, but it would be good to make fp16 as a separate module to reduce duplicate codes with souschef.

**Review by jinevening — APPROVED** [review]
LGTM Thank you

**Pull request merged**